### PR TITLE
Fix/registrykey

### PR DESCRIPTION
**Pull request by Raycoms**
# Changes proposed in this pull request:
- Fix a bit of the registrykey mess we're having with the dimensiontypes
- Also fix the capability mess
-

Review please

### CODE REVIEW

**Review by marchermans — DISMISSED** [review]

File: src/main/java/com/minecolonies/coremod/network/messages/splitting/SplitPacketMessage.java [hunk 1]
```diff
@@ -111,20 +112,21 @@ public void onExecute(final NetworkEvent.Context ctxIn, final boolean isLogicalS
...
             buffer.release();
 
             //Execute the message.
-            final LogicalSide packetOrigin = ctxIn.getDirection().getOriginationSide();
-            if (message.getExecutionSide() != null && packetOrigin.equals(message.getExecutionSide()))
-            {
-                Log.getLogger().warn("Receving {} at wrong side!", message.getClass().getName());
-                return;
-            }
-            // boolean param MUST equals true if packet arrived at logical server
```
Comment (Nightenom) [inline src/main/java/com/minecolonies/coremod/network/messages/splitting/SplitPacketMessage.java]: comment move

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/network/messages/splitting/SplitPacketMessage.java]: I undid this afaik

File: src/main/java/com/minecolonies/coremod/util/BackUpHelper.java [hunk 2]
```diff
@@ -305,19 +312,20 @@ public static void loadAllBackups()
...
         @NotNull final File saveDir = new File(ServerLifecycleHooks.getCurrentServer().func_240776_a_(FolderName.DOT).toFile(), FILENAME_MINECOLONIES_PATH);
-        CompoundNBT compound = loadNBTFromPath(new File(saveDir, String.format(FILENAME_COLONY, colonyId, dimension)));
+        @NotNull final File backupFile = new File(saveDir, String.format(FILENAME_COLONY, colonyId, dimension.getLocation()));
+        CompoundNBT compound = loadNBTFromPath(backupFile);
         if (compound == null)
         {
             if (loadDeleted)
             {
-                compound = loadNBTFromPath(new File(saveDir, String.format(FILENAME_COLONY_DELETED, colonyId, dimension)));
+                compound = loadNBTFromPath(backupFile);
```
Comment (someaddons) [inline src/main/java/com/minecolonies/coremod/util/BackUpHelper.java:324]: revert this, was there for a reason namely loading .deleted file colony backups

File: src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingBaker.java [hunk 3]
```diff
@@ -157,32 +132,6 @@ protected boolean keepFood()
         return new JobBaker(citizen);
     }
 
-    @Override
-    public void deserializeNBT(final CompoundNBT compound)
```
Comment (someaddons) [inline src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingBaker.java:161]: was this stuff just unused? if not probably should port that to 1.15 aswell

**Pull request merged**